Tests/test_Seq_objs.py

```diff
@@ -737,6 +737,27 @@ def test_init_typeerror(self):
         self.assertRaises(TypeError, Seq, (1066))
         self.assertRaises(TypeError, Seq, (Seq("ACGT", generic_dna)))
 
+    def test_join(self):
+        """Checks if Seq join correctly concatinates sequence with the spacer."""
+        # Only expect it to take Seq objects only!
```

Comment:
No, we also expect strings (although if you mean in this test, make that clearer)